tests/create_app.go

```diff
@@ -65,6 +65,6 @@ func CreateLegacyApp(svc *TestService, senderPrivkey string) (app *db.App, ss []
 		},
 	})
 
-	ss, err = nip04.ComputeSharedSecret(svc.Keys.GetNostrPublicKey(), senderPrivkey)
-	return app, ss, nil
+	nip47Cipher, err = cipher.NewNip47Cipher("1.0", svc.Keys.GetNostrPublicKey(), senderPrivkey)
```

Comment:
Should legacy app return a cipher with v0? Or should we add `CreateAppV0` and `CreateAppV1` 😵‍💫

Reply:
Can you rename the function to CreateAppWithoutWalletPubkey and add a `nip47Version` argument?

Reply:
Done